doc/api/worker_threads.md

```diff
@@ -679,6 +679,29 @@ The `ArrayBuffer`s for `Buffer` instances created using
 transferred but doing so renders all other existing views of
 those `ArrayBuffer`s unusable.
 
+#### Synchronous blocking of message handling
+
+The `postMessage()` method will defer the actual transmission and
+handling of the message until after the current synchronous execution
+context completes and the Node.js event loop as allowed to continue
+turning.
```

Comment:
This is – observably, we have tests for it – not true, `.postMessage()` does work fully synchronously.